app/components/UI/Ramp/Deposit/Views/BuildQuote/BuildQuote.tsx

```diff
@@ -12,26 +14,50 @@ import { useDepositSdkMethod } from '../../hooks/useDepositSdkMethod';
 import { createProviderWebviewNavDetails } from '../ProviderWebview/ProviderWebview';
 import { createBasicInfoNavDetails } from '../BasicInfo/BasicInfo';
 import { createEnterEmailNavDetails } from '../EnterEmail/EnterEmail';
-import { View } from 'react-native';
-import DepositTextField from '../../components/DepositTextField';
+import { View, TouchableOpacity, Image } from 'react-native';
+import Keypad from '../../../../../Base/Keypad';
+import Icon, {
+  IconName,
+  IconSize,
+} from '../../../../../../component-library/components/Icons/Icon';
+import {
+  DEBIT_CREDIT_PAYMENT_METHOD,
+  USDC_TOKEN,
+  DepositCryptoCurrency,
+  DepositPaymentMethod,
+  USD_CURRENCY,
+  DepositFiatCurrency,
+} from '../../constants';
+import AccountSelector from '../../components/AccountSelector';
+import { strings } from '../../../../../../../locales/i18n';
 
 const BuildQuote = () => {
   const navigation = useNavigation();
+
   const { styles, theme } = useStyles(styleSheet, {});
 
-  const [paymentMethod] = useState<string>('credit_debit_card');
-  const [cryptoCurrency] = useState<string>('USDC');
-  const [fiatCurrency] = useState<string>('USD');
+  const [paymentMethod, setPaymentMethod] = useState<DepositPaymentMethod>(
```

Comment:
```suggestion
  const [paymentMethod] = useState<DepositPaymentMethod>(
```

I think the setter can be removed for now to fix `'setPaymentMethod' is declared but its value is never read.ts(6133)`

Reply:
Same with `setCryptoCurrency`, `setFiatCurrency`